charts/prometheus-statsd-exporter/values.yaml

```diff
@@ -85,9 +85,9 @@ ingress:
   annotations: {}
     # kubernetes.io/ingress.class: nginx
     # kubernetes.io/tls-acme: "true"
+  # pathType: ImplementationSpecific # only Kubernetes v1.19+
```

Comment:
Imho wron place for this. The ingress default config for a helm 3.6.0 created shard would look like:

```
ingress:
  enabled: false
  className: ""
  annotations: {}
    # kubernetes.io/ingress.class: nginx
    # kubernetes.io/tls-acme: "true"
  hosts:
    - host: chart-example.local
      paths:
        - path: /
          pathType: ImplementationSpecific
  tls: []
  #  - secretName: chart-example-tls
  #    hosts:
  #      - chart-example.local
```